modules/renter/contractor/contractor_test.go

```diff
@@ -198,6 +198,114 @@ func (stubHostDB) ScoreBreakdown(modules.HostDBEntry) modules.HostScoreBreakdown
 	return modules.HostScoreBreakdown{}
 }
 
+// TestAllowanceOverspend verifies that the contractor will not spend more
+// than the allowance if contracts need to be renewed early.
+func TestAllowanceOverspend(t *testing.T) {
+	t.Skip("FIXME: test requires better money management to pass")
```

Comment:
is the idea that this will be un-skipped after a future PR is merged?

Reply:
Yeah, I think right now we don't actually have controls to prevent the allowance from over spending. That'll be my next PR after this is merged.